src/main/java/li/cil/scannable/common/item/ScannerItem.java

```diff
@@ -178,16 +164,18 @@ public ItemStack finishUsingItem(final ItemStack stack, final Level level, final
             return stack;
         }
 
+        /*
         if (level.isClientSide()) {
             SoundCanceler.cancelEquipSound();
         }
+         */
```

Comment:
Is this workaround not (no longer?) needed, or did it not work for Fabric?

Reply:
I'm not sure the equivalent event exists in Fabric (though a mixin could add it relatively easily), and I don't seem to hear any reequip sound or see anything in the subtitles when the scan finishes.

Reply:
Just tested a bit, and doesn't seem to happen anymore, so this can be completely removed 🎉

The issue this was added for was, that in non-creative, when energy stored changed, the NBT changed, the client was notified of that change, the client thought it was a new item, and played the equip sound. Looks like that's no longer necessary, so yay.